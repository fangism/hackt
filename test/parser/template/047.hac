// "047.hac" should fail, no default relaxed parameters
// copy-modified from "044.hac"

template <pint Q=5><pint R>
defproc foo();

foo<><> bar;	// should die, insufficient default parameters

