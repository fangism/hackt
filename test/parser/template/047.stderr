Error: template parameter is required but missing at position 1. 
ERROR: failed to make process_type_reference because template argument types do not match.
ERROR making complete type reference.  [7:1..7]
